man/dnfdragora.rst

```diff
@@ -0,0 +1,99 @@
+:orphan:
+
+..
+  Copyright (C) 2016-2017 Angelo Naselli and Neal Gompa
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+.. _command_ref-label:
+
+##############################
+ DNFDragora Command Reference
+##############################
+
+==========
+ Synopsis
+==========
+
+``dnfdragora [options] [<args>...]``
+
+=============
+ Description
+=============
+
+.. _command_provides-label:
+
+DNFDragora a is a DNF frontend, based on rpmdragora from Mageia
```

Comment:
I get that this is generated, so where is this wrongly structured sentence coming from? It should be fixed.